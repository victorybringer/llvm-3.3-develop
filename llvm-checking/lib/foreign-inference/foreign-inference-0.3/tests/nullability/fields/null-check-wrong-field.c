struct Pair {
  int *f1;
  int *f2;
};

int g;

void notNullableField(struct Pair *p) {
  if(p->f1)
    g = *p->f2;
}
