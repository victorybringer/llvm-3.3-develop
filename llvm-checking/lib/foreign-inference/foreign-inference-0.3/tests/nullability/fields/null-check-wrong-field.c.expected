fromList [("%struct.Pair", 1)]
